### ISSUE TITLE
Bug: vocals reset after death

### ISSUE THREAD

**Issue by CavernKeeper** [AbnormalPoof/FunkerSelector#3]
### Version

0.4.1

### Issue Checklist

- [X] Have you checked the Issues/Discussions to see if this hasn't been reported yet?
- [X] Have you tried reinstalling the mod?
- [X] Is this a genuine issue with Funker Selector and not a side-effect of another mod you have installed?

### Context (Provide images, videos, etc.)

I commented this issue earlier before reading the FAQ lol my bad. 

Same issue, when playing a character with their own seperate vocal track and dying/getting blueballed, the vocal track resets to default (or bf's vocals)

I assume this is an issue with whatever was refactored with how the vocals are replaced, the older version of the mod doesn't have this problem.

**Comment by AbnormalPoof** [comment]
Does commit [5321352](https://github.com/AbnormalPoof/FunkerSelector/commit/5321352fde44f7aae865c5e922afaf269becb858) fix the issue?

**Comment by CavernKeeper** [comment]
yeah

i also get these error messages when opening the character selector menu thing, not sure if thats my fault or smthn just happens
![2024-08-12 (2)](https://github.com/user-attachments/assets/8c63f02f-13ed-4cbd-a736-aa495bd399b3)
![2024-08-12 (3)](https://github.com/user-attachments/assets/01c48007-446f-457d-ad06-3a9f38ad1ae7)

**Comment by CavernKeeper** [comment]
i wasnt sure what to do w this so i just downloaded the files n threw them in the mod lol
idk what i was meant to do mb

**Comment by AbnormalPoof** [comment]
This means it failed to initialize the substate, it appears to be trying to load `CharacterMenu`. I changed the class names a bit ago. Are you sure you downloaded the right one? This line in `charHandler` (or charSelect if you downloaded the wrong one) should be:

```haxe
var charSelect:FlxSubState = ScriptedMusicBeatSubState.init("CharacterSelectSubState");
```

**Comment by CavernKeeper** [comment]
loll I'm dumb i put them in the wrong folders. yeah everything's good now thank u

**Comment by AbnormalPoof** [comment]
Fixed in v1.2.2.
